static/src/javascripts/projects/common/modules/ui/cmp-ui.js

```diff
@@ -81,12 +45,11 @@ export const addPrivacySettingsLink = (): void => {
 export const consentManagementPlatformUi = {
     id: 'cmpUi',
     canShow: (): Promise<boolean> => {
-        if (shouldUseSourcepointCmp()) {
-            return Promise.resolve(cmp.willShowPrivacyMessage());
-        }
-        return Promise.resolve(
-            config.get('switches.cmpUi', true) && oldCmp.shouldShow()
-        );
+        if (!config.get('switches.cmp', true)) return Promise.resolve(false);
+        return Promise.resolve(cmp.willShowPrivacyMessage());
+    },
+    show: (): Promise<boolean> => {
```

Comment:
why is this a promise? because the previous implementation was?

Reply:
Yes. I will create a deeper clean PR following this one.